app/src/pages/playground/__tests__/playgroundUtils.test.ts

```diff
@@ -303,6 +303,36 @@ describe("transformSpanAttributesToPlaygroundInstance", () => {
       parsingErrors: [],
     });
   });
+
+  it("should correctly parse the invocation parameters", () => {
+    const span = {
+      ...basePlaygroundSpan,
+      attributes: JSON.stringify({
+        ...spanAttributesWithInputMessages,
+        llm: {
+          ...spanAttributesWithInputMessages.llm,
+          // note that camel case keys are automatically converted to snake case
+          invocation_parameters:
+            '{"topP": 0.5, "maxTokens": 100, "seed": 12345, "stop": ["stop", "me"]}',
+        },
+      }),
+    };
```

Comment:
wait which direction it goes snake to camel right? can we get a test showing that with at least one param as well, c just convert one of these params

Reply:
Updated comment and test, cursor got me mixed up 😂